playbooks/common/openshift-cluster/redeploy-certificates/ca.yml

```diff
@@ -9,7 +9,8 @@
 - name: Backup existing etcd CA certificate directories
   hosts: oo_etcd_to_config
   roles:
-  - etcd_common
+  - role: etcd_common
+    r_etcd_common_etcd_runtime: "{{ 'runc' if openshift.common.is_etcd_system_container | bool else 'docker' if openshift.common.is_containerized | bool else 'host' }}"
```

Comment:
This line is used a lot. Is there a way to set it once and use it? Or does it need to be created each time?